identity-server/src/IdentityServer/Configuration/DependencyInjection/Options/PreviewFeaturesOptions.cs

```diff
@@ -8,6 +10,7 @@ public class PreviewFeaturesOptions
     /// <summary>
     /// Enables Caching of Discovery Document based on ResponseCaching Interval 
     /// </summary>
+    [Experimental("DUENDEPREVIEW001", UrlFormat = "https://duende.link/previewfeatures?id={0}")]
```

Comment:
Who is building this endpoint? 😅

Reply:
Easy!

https://github.com/DuendeSoftware/fenix/pull/5
https://github.com/DuendeSoftware/docs.duendesoftware.com/pull/611